src/components/Layout/NestedScrollLayoutContext.tsx

```diff
@@ -0,0 +1,38 @@
+import { createContext, PropsWithChildren, useEffect, useMemo, useRef } from "react";
+import { Css, Only, Xss } from "src/Css";
+
+interface NestedScrollLayoutContextProps {
+  scrollableEl: HTMLElement;
+}
+
+export const NestedScrollLayoutContext = createContext<NestedScrollLayoutContextProps>({
+  scrollableEl: document.createElement("div"),
```

Comment:
Wondering if this really needs to be reset in the `NestedScrollLayoutContextProvider` function?